src/modules/420_graph_out.rs

```diff
@@ -454,3 +497,62 @@ fn create_swap_entity(tables: &mut Tables, event: &PoolEvent, swap: &SwapEvent)
         .set("amountOutUSD", BigDecimal::zero())
         .set("pool", format::format_address_string(&event.pool_address));
 }
+
+fn create_swap_underlying_entity(
+    tables: &mut Tables,
+    event: &PoolEvent,
+    swap_underlying: &SwapUnderlyingEvent,
+) {
+    let key = format!("swap-0x{}-{}", event.transaction_hash, event.log_index);
+    tables
+        .create_row("Swap", key)
+        .set(
+            "hash",
+            format::format_address_string(&event.transaction_hash),
+        )
+        .set("logIndex", event.log_index as i32)
+        .set("protocol", utils::get_protocol_id())
+        .set("to", format::format_address_string(&event.to_address))
+        .set("from", format::format_address_string(&event.from_address))
+        .set("blockNumber", BigInt::from(event.block_number))
```

Comment:
If these are Strings you can use .set_bigint instead of .set

Reply:
They were `u64`'s so kept as set for now.